_includes/helpers.js

```diff
@@ -0,0 +1,141 @@
+function buildTable(insideNode, confs, includeCFP) {
+	var table = document.createElement("table");
+	insideNode.appendChild(table);
+	var thead = document.createElement("thead");
+	table.appendChild(thead);
+	var headRow = document.createElement("tr");
+	thead.appendChild(headRow);
+	
+	var nameHead = document.createElement("th");
+	nameHead.appendChild(document.createTextNode("Name"));
+	headRow.appendChild(nameHead);
+	var dateHead = document.createElement("th");
+	dateHead.appendChild(document.createTextNode("Date"));
+	headRow.appendChild(dateHead);
+	var placeHead = document.createElement("th");
+	placeHead.appendChild(document.createTextNode("Place"));
+	headRow.appendChild(placeHead);
+
+	if (includeCFP === true) {
+		var cfpHead = document.createElement("th");
+		cfpHead.appendChild(document.createTextNode("CFP Deadline"));
+		headRow.appendChild(cfpHead);
+	}
+	
+	var tbody = document.createElement("tbody");
+	table.appendChild(tbody);
+	
+	for (var i = 0; i < confs.length; i++) {
+		tbody.appendChild(buildRow(confs[i], includeCFP));
+	}
+
+}
+
+var months = ["", "January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
```

Comment:
I think this, and some of your other vars, can be `const` instead of `var`.  (think about Swift var vs. let. Javascript's let is mutable, for some ungodly reason)

Reply:
Oh, I had no idea `const` was a thing.